packages/webapp/src/components/Modals/WaterUseCalculatorModal/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import ModalComponent from '../ModalComponent/v2';
+import { ReactComponent as Person } from '../../../assets/images/task/Person.svg';
+import styles from '../QuickAssignModal/styles.module.scss';
+import Button from '../../Form/Button';
+
+export interface IWaterUseCalculatorModal {
+  dismissModal: () => void;
+  // setTotalWaterUsage: () => number,
+}
+
+const WaterUseCalculatorModal: FC<IWaterUseCalculatorModal> = ({ dismissModal }) => {
```

Comment:
Add propTypes

Reply:
@anant00729 with typescript, propTypes aren't required. Both typescript types/interfaces work the same as propTypes.

Reply:
got it. thanks!